projects/editions/pkg/install-dfm/Makefile

```diff
@@ -0,0 +1,33 @@
+default=tag
+ORG?=edtestorg
+IMAGE=install-dfm
+DEPS=Dockerfile build/dfm/port build/dfm/docker
+
+HASH?=$(shell git ls-tree HEAD -- ../$(notdir $(CURDIR)) | awk '{print $$3}')
+
+# Create empty dir for /port mount
+build/dfm/port: Makefile
+	mkdir -p build/dfm/port
+
+# Configure initial docker dir, install memlogd plugin
+build/dfm/docker: Makefile
+	mkdir -p build/dfm/docker/plugins
+	docker run --rm --privileged \
+		-v $(PWD)/build/dfm/docker:/var/lib/docker \
+		--name $(IMAGE)-dind \
+		-d docker:17.05.0-ce-dind \
+		--storage-driver=overlay2
+	docker run --rm --link $(IMAGE)-dind:docker docker:edge plugin ls | grep -q memlogd-docker || ( \
+		docker run --rm --link $(IMAGE)-dind:docker docker:edge plugin install cpuguy83/memlogd-docker --grant-all-permissions || ( \
+			docker stop $(IMAGE)-dind ; exit 1 ) )
+	docker stop $(IMAGE)-dind
```

Comment:
We have an issue open for how to handle plugins nicely. The vast majority of people want plugins just installed once not dynamically, so it would be nice if we could support that properly.